src/mantine-core/src/Rating/Rating.tsx

```diff
@@ -0,0 +1,302 @@
+import React, { forwardRef, useId, useState, useRef } from 'react';
+import { DefaultProps, MantineSize, useComponentDefaultProps, Selectors } from '@mantine/styles';
+import { useUncontrolled, mergeRefs, clamp } from '@mantine/hooks';
+import { StarSymbol } from './StarSymbol';
+import { Box } from '../Box';
+import useStyles, { RatingStylesParams } from './Rating.styles';
+import { roundValueTo } from './utils';
+
+interface SymbolProps {
```

Comment:
Extract Symbol component to a separate file, also rename it – Symbol is a built-in constructor